### DISCUSSION TITLE
Using Fish Market

### DISCUSSION THREAD

**Discussion by steffink25** [Tunny-gh/Tunny_in_Open_Sea#168] [General]
I am struggling to get Fish Market to work. I am somewhat unsure of what is necessary to make it run. In one attempt, I tried to plug in a geometry attribute and here is my canvas, but the range of various shapes does not array. I did run Tunny's Optimization.
![image](https://user-images.githubusercontent.com/129544367/229197088-f9ed2ecd-2d12-4186-83c8-e9d866357c5b.png)

I tried running Tunny without the attributes and it doesn't generate optimized fishes now.
![image](https://user-images.githubusercontent.com/129544367/229200737-e86533e1-8b18-4a6b-a237-85677a80bdd4.png)

The Tunny docs page is helpful, but are there sample .gh files of working full Tunny set ups available to help navigate the tool? In following the steps in the video at the bottom of the discussion page, when I click "pareto solutions" it just stalls and never finishes.

**Comment by hrntsm** [comment]
Hi @steffink25, Thanks for using tunny.

If you are downloading from food4rhino, the file sample.gh is included, so look at that file.
Tunny is not included if you have installed it from PackageManager, so use the file attached.
[Samples.zip](https://github.com/hrntsm/Tunny/files/11128464/Samples.zip)

If the optimisation has been carried out without problems, the results should be output by selecting the target from the TargetStudy and pressing a button such as "Pareto solutions", as shown below.
<img width="565" alt="Screenshot 2023-04-01 at 13 58 43" src="https://user-images.githubusercontent.com/23289252/229266466-bd8bb1d1-19a4-4298-9962-1fcb391f54f9.png">

If possible, please provide us with the gh file so it can be checked to see what is causing the problem.
